sendmarc.com.protection.json

```diff
@@ -0,0 +1,103 @@
+{
+  "providerId": "sendmarc.com",
+  "providerName": "Sendmarc",
+  "serviceId": "protection",
+  "serviceName": "Sendmarc Email Security",
+  "version": 1,
+  "syncRedirectDomain": "sendmarc.com",
+  "logoUrl": "https://portal.sendmarc.com/img/light-bg-transparent.svg",
+  "syncPubKeyDomain": "domainconnect.sendmarc.com",
+  "warnPhishing": true,
+  "description": "Stop cyber-criminals from sending email using your domain and protect your customers, suppliers and employees from attacks.",
+  "variableDescription": "Self explanatory in the variable names",
+  "records": [
+    {
+      "groupId": "dmarc",
+      "type": "CNAME",
+      "host": "_dmarc",
+      "pointsTo": "%dmarc%",
+      "ttl": 3600
+    },
+    {
+      "groupId": "spf",
+      "type": "SPFM",
+      "host": "@",
+      "spfRules": "$spf_rules%"
```

Comment:
I assume it should have been `%spf_rules%`

Reply:
Yes sorry about that. I made a typo